python/paddle/v2/dataset/imikolov.py

```diff
@@ -41,7 +41,7 @@ def word_count(f, word_freq=None):
     return word_freq
 
 
-def build_dict():
+def build_dict(TYPO_FREQ=50):
```

Comment:
If  `TYPO_FREQ` is passed to the function as a parameter, a lower case should be used.